activerecord/lib/active_record/railtie.rb

```diff
@@ -108,8 +109,11 @@ class Railtie < Rails::Railtie # :nodoc:
 
     initializer "active_record.set_configs" do |app|
       ActiveSupport.on_load(:active_record) do
-        app.config.active_record.each do |k, v|
-          send "#{k}=", v
+        configs = app.config.active_record.dup
+        configs.delete(:sqlite3_represent_boolean_as_integer)
```

Comment:
Is this because of the assignment above? The need for this seems odd to me, and I think we should avoid it if we can.